src/bin/ccg.rs

```diff
@@ -0,0 +1,393 @@
+#[macro_use]
+extern crate log;
+
+use bob::configs::cluster::{Config, Node as ClusterNode, Replica, VDisk};
+use chrono::Local;
+use clap::{App, Arg, ArgMatches};
+use env_logger::fmt::Color;
+use log::{Level, LevelFilter};
+use std::fs::{File, OpenOptions};
+use std::io::{Read, Write};
+use std::sync::atomic::{AtomicUsize, Ordering};
+
+const ORD: Ordering = Ordering::Relaxed;
+
+#[tokio::main]
+async fn main() {
+    init_logger();
+    if let Some(output) = read_from_file().and_then(generate_config) {
```

Comment:
it would be nice to have option to output to stdout or have only option to output to stdout. then it could be used as `./ccg --inut ttt > output.cfg`

Reply:
Done.